doc/docker-compose-to-toolkit-migration.md

```diff
@@ -0,0 +1,78 @@
+# docker-compose.yml to Toolkit migration #
+
+If you're currently using Docker Compose via a `docker-compose.yml` file, migrating to the Toolkit can make running an on-premises version of Overleaf easier to deploy, upgrade and maintain.
+
+To upgrade, you'll need to convert your existing Docker Compose setup into the format used by the Toolkit. This process involves copying existing configuration into the Toolkit.
+
+This guide will walk you through each step of this process, ensuring a smooth migration from Docker Compose to the Toolkit.
+
+**Note:** These instructions are for v4.x and earlier. Therefore all variables use the `SHARELATEX_` prefix instead of `OVERLEAF_`
+
+## Clone the Toolkit repository ##
+
+First, let's clone this Toolkit repository to the host machine:
+```
+$ git clone https://github.com/overleaf/toolkit.git ./overleaf-toolkit
+```
+Next run the `bin/init` command to initialise the Toolkit with its default configuration.
+
+## Setting the image and version ##
+
+In the `docker-compose.yml` file the image and version are defined in the component description:
+
+```
+version: '2.2'
+services:
+    sharelatex:
+        restart: always
+        # Server Pro users:
+        # image: quay.io/sharelatex/sharelatex-pro
+        image: sharelatex/sharelatex:3.5.13
+```
+
+When using the Toolkit, the image name is automatically resolved; the only requirement is to set `SERVER_PRO=true` in **config/overleaf.rc** to pick the Server Pro image or `SERVER_PRO=false` to use Community Edition.
+
+The desired version number is set in the **config/version** file. We recommend avoiding the use of "**latest**", and instead using a specific version number like "**4.2.3**".
```

Comment:
```
toolkit$  echo latest > config/version 
toolkit$  bin/docker-compose ps
ERROR: invalid version 'latest'
```

They cannot set `latest` :)

```suggestion
The desired Server Pro/Community Edition version number is set in the **config/version** file. The Toolkit requires a specific version number like "**4.2.3**". In case you are using `latest`, you can use `bin/images` to find the image id of your local `latest` version, then use the release notes for [2.x](https://github.com/overleaf/overleaf/wiki/Release-Notes-2.0), [3.x](https://github.com/overleaf/overleaf/wiki/Release-Notes-3.x.x), [4.x](https://github.com/overleaf/overleaf/wiki/Release-Notes--4.x.x) or [5.x](https://github.com/overleaf/overleaf/wiki/Release-Notes-5.x.x) to map the image id to the version.
```

(https://github.com/overleaf/toolkit/pull/234 will make the output of `bin/images` easier to read.)